cmd/plc-server/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/stellentus/go-plc"
+)
+
+var (
+	plcAddr  = flag.String("plc-address", "192.168.1.176", "Hostname or IP address of the PLC")
+	path     = flag.String("path", "1,0", "Path to the PLC at the provided host or IP")
+	httpAddr = flag.String("http", ":8784", "Port for http server to listen to")
+)
+
+var knownTags = map[string]interface{}{
+	"Program:ZW.B801.INISCP":      uint32(0),
```

Comment:
Should these known tags be added to plcstub's default tag list?

Reply:
I'd rather not see these names committed to this repo. I'd prefer to add a flag with a CSV list of known tags, or a path to a file with these names. And the default in this repo should use `DUMMY_AQUA_DATA_0[0]`, etc.

Reply:
As I think about this more, I remember it's not so easy to load this because they need to be typed. I have some ideas, but I'll reflect on it more before proposing anything. (Um, no pun intended, but it's certainly relevant to consider reflecting on this.)